t04_refactored.py

```diff
@@ -137,6 +137,66 @@ def team_3_adv():
 def team_4_adv():
     pass
     # TODO Add your code here
+    # """
+    # https://docs.google.com/document/d/1BT_3NDURoZ_V-Ztx1OQcLPjn5krxuLOk8cLPmETCF2c/edit
+    # Kafui Gle
+    # Samundra Adhikari
+    # :return: none
+    # """
```

Comment:
The docstring shouldn't be commented out.

Reply:
Docstring is no  longer commented out.